### ISSUE TITLE
./freeze.py for updating base environments fails

### ISSUE THREAD

**Issue by yuvipanda** [jupyterhub/repo2docker#1037]
### Bug description

When running `./freeze.py` under `repo2docker/buidpacks/conda`, I get the following error:

```
╰─$ ./freeze.py                                                                                                           130 ↵
Freezing /Users/yuvipanda/code/repo2docker/repo2docker/buildpacks/conda/environment.py-2.7.yml -> /Users/yuvipanda/code/repo2docker/repo2docker/buildpacks/conda/environment.py-2.7.frozen.yml
+ conda config --set channel_priority strict
+ conda config --add channels conda-forge
+ true
+ true
+ conda config --system --set auto_update_conda false
+ mamba env create -v -f /r2d/environment.py-2.7.yml -n r2d
Traceback (most recent call last):
  File "/opt/conda/lib/python3.8/site-packages/conda/exceptions.py", line 1079, in __call__
    return func(*args, **kwargs)
  File "/opt/conda/lib/python3.8/site-packages/conda_env/cli/main.py", line 80, in do_call
    exit_code = getattr(module, func_name)(args, parser)
  File "/opt/conda/lib/python3.8/site-packages/conda_env/cli/main_create.py", line 87, in execute
    spec = specs.detect(name=name, filename=filename, directory=os.getcwd())
  File "/opt/conda/lib/python3.8/site-packages/conda_env/specs/__init__.py", line 49, in detect
    raise SpecNotFound(build_message(spec_instances))
conda_env.exceptions.SpecNotFound: Invalid name, try the format: user/package

Traceback (most recent call last):
  File "/Users/yuvipanda/code/repo2docker/repo2docker/buildpacks/conda/./freeze.py", line 135, in <module>
    freeze(env_file, frozen_file)
  File "/Users/yuvipanda/code/repo2docker/repo2docker/buildpacks/conda/./freeze.py", line 67, in freeze
    check_call(
  File "/usr/local/Cellar/python@3.9/3.9.1_6/Frameworks/Python.framework/Versions/3.9/lib/python3.9/subprocess.py", line 373, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['docker', 'run', '--rm', '-v/Users/yuvipanda/code/repo2docker/repo2docker/buildpacks/conda:/r2d', '-it', 'condaforge/mambaforge:4.9.2-5', 'sh', '-c', 'set -ex; conda config --set channel_priority strict; conda config --add channels conda-forge; true; true; conda config --system --set auto_update_conda false; mamba env create -v -f /r2d/environment.py-2.7.yml -n r2d; conda config --append channels conda-forge/label/broken; mamba env export -n r2d >> /r2d/environment.py-2.7.frozen.yml']' returned non-zero exit status 1.
```

#### Expected behaviour

It should regenerate the various environment.yml files for each python version.

### How to reproduce

1. Clone this repo
2. Run `./repo2docker/buildpacks/conda/freeze.py`

### Your personal set up

Clone of current master

**Comment by manics** [comment]
There's a PR to replace freeze.py in https://github.com/jupyterhub/repo2docker/pull/1024

**Comment by consideRatio** [comment]
Closing, assumed to be resolved by #1024
